main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	//"encoding/json"
+	//"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/ini8labs/sns/src/apis"
```

Comment:
use proper import format:
1. built-in libraries.
2. 3rd party libraries
3. project package/libraries